cli.js

```diff
@@ -22,6 +22,10 @@ if (!args.dir || (!args.all && (!args.platform || !args.arch))) {
   process.exit(1)
 }
 
+if (args.tmpdir === 'false') {
+  args.tmpdir = false
+}
+
```

Comment:
Please also add documentation to `readme.md` and `usage.txt` about this new functionality to `tmpdir`. Though you may want to hold off on that until #275 is merged.